### PR TITLE
✨ Add DASes (#49)

### PR DESCRIPTION
**Pull request by mistryrn**
* Add `consent-das`, `keys-das`, `phi-das`, and `pi-das` to `apps/` dir
* Add `docker-scripts/` dir, containing Docker init script to enable multiple postgres dbs in one container
* Update root `.gitignore` to ignore generated Prisma client `generated` output dirs
* Update `docker-compose.yaml` to set up multiple databases, one per DAS
* Update root `package.json` with new scripts: `das-dev` and `seed` for the DASes
* Modify existing scripts in root `package.json`: `build`, `dev`, `migrate-dev`, `generate`, and `start` to account for new changes in the repo due to the DASes

### CODE REVIEW

File: apps/consent-das/src/server.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	const appConfig = getAppConfig();
+	logger.info('Initializing server.ts');
+
+	const app = App(appConfig);
+	const port = app.get('port');
+	server = app.listen(port, () => {
+		logger.info(`Server listening on port ${port}`);
+	});
+
+	process.on('SIGINT', () => {
```
Comment (anncatton) [inline apps/consent-das/src/server.ts:40]: 🙏

**Review by anncatton — APPROVED** [review]
Looks great, this will be such a good foundation for all the feature implementation...thanks for all the TODOs!
Remind me, were we going to add todos for the swagger/jsdoc setup too?

**Comment by mistryrn** [comment]
> Remind me, were we going to add todos for the swagger/jsdoc setup too?

The Swagger/JSdoc stuff was meant to be a ticket, thanks for the reminder! I'll make that before merging 🙏

**Comment by mistryrn** [comment]
@anncatton Swagger/JSDoc ticket has been created: https://github.com/OHCRN/platform/issues/70

**Pull request merged**